### PR TITLE
Add perl-uri-query new recipe

### PR DESCRIPTION
**Pull request by fgiacomoni**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by fgiacomoni** [comment]
@conda-forge-admin we need a review here - Thank you

**Comment by fgiacomoni** [comment]
Dear @xileF1337  have you a minute to review this dependency. Big Thanks

**Comment by fgiacomoni** [comment]
@conda-forge-admin 
Any help here... my "to the production pipeline" stage is totally blocked...

**Comment by fgiacomoni** [comment]
Hi @xileF1337 sorry for the insisting ping. I'm (really) late now to deliver a tool under galaxy for a national project. Is it possible for you to review my recipe? Thank you very much in advance for the effort and the time.

**Comment by fgiacomoni** [comment]
No movement on this side before holiday????????? Eternal thanks to @xileF1337 or @conda-forge/help-perl

File: recipes/perl-uri-query/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    weak:
+      - {{ name }} ={{ version }}
+
+requirements:
+  build:
+        - make
```
Comment (xileF1337) [inline recipes/perl-uri-query/meta.yaml:22]: Please fix the indentation here.

**Review by xileF1337 — CHANGES_REQUESTED** [review]
Looks very good to me. If you could fix the indentation of that single line below, the PR could be merged IMO.

**Comment by xileF1337** [comment]
Hi @fgiacomoni, I'm incredibly sorry it took so long to take a look at your PR, but I quit academia a while ago and my time for maintaining Conda packages is rather limited now. I hope we can get this done soon.

File: recipes/perl-uri-query/meta.yaml (hunk 1 above)
Comment (fgiacomoni) [reply recipes/perl-uri-query/meta.yaml]: Done! Thanks @xileF1337 (Sorry for the delay, back from holidays...)

**Comment by fgiacomoni** [comment]
Project stuck... Eternal thanks to @xileF1337  or @conda-forge/help-perl Any body to make me progress in my package?

**Comment by fgiacomoni** [comment]
@bgruening sorry for the invocation - but do you know someone expert in Perl and getting merge rights in conda-forge. Big Thanks

**Comment by fgiacomoni** [comment]
@conda-forge-admin is there any possibility to accept my reviewed PR by @xileF1337? Big thanks

**Comment by fgiacomoni** [comment]
All tests passed if any reviewers (@beckermr or @cbrueffer ??) can merge my PR - big big thanks

**Review by xileF1337 — APPROVED** [review]

**Comment by xileF1337** [comment]
@bgruening could you consider merging this PR, please? Thank you so much!

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks all!

**Comment by xileF1337** [comment]
Sorry again that it took so long!

**Pull request merged**